billy-france/src/test/java/com/premiumminds/billy/france/test/services/export/TestFRCreditNotePDFTransformer.java

```diff
@@ -118,28 +121,35 @@ public void testNonExistentEntity() {
     }
 
     @Test
-    public void testNonExistentInvoice() throws DocumentIssuingException {
+    public void testNonExistentInvoice()
+		throws DocumentIssuingException, SeriesUniqueCodeNotFilled, DocumentSeriesDoesNotExistException
+	{
 
         UID uidEntity = UID.fromString("12345");
         final String businessUID = (new UID()).toString();
         this.createSeries(businessUID);
         FRInvoiceEntity invoice = this.getNewIssuedInvoice(businessUID);
-        FRCreditNoteEntity entity = this.generateFRCreditNote(PaymentMechanism.CASH, invoice);
+        FRCreditNoteEntity entity = this.generateFRCreditNote(invoice);
         DAOFRCreditNote dao = this.mockedInjector.getInstance(DAOFRCreditNote.class);
         Mockito.when(dao.get(ArgumentMatchers.eq(uidEntity))).thenReturn(entity);
 
         Assertions.assertThrows(ExportServiceException.class, () -> this.extractor.extract(uidEntity));
     }
 
     @Test
-    public void testPdfCreationFromBundle() throws ExportServiceException,
-            DocumentIssuingException, IOException {
+    public void testPdfCreationFromBundle() throws
+											ExportServiceException,
+											DocumentIssuingException,
+											IOException,
+											SeriesUniqueCodeNotFilled,
+											DocumentSeriesDoesNotExistException
```

Comment:
This portion of code doesn't follow the project guidelines. Make sure you use the given formatter that follows the project guidelines